gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/test/java/inmemory/IntegrationQueryServiceInMemory.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright © 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package inmemory;
+
+import io.gravitee.apim.core.integration.model.Integration;
+import io.gravitee.apim.core.integration.query_service.IntegrationQueryService;
+import io.gravitee.common.data.domain.Page;
+import io.gravitee.rest.api.model.common.Pageable;
+import io.gravitee.rest.api.service.common.UuidString;
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+public class IntegrationQueryServiceInMemory implements IntegrationQueryService, InMemoryAlternative<Integration> {
```

Comment:
As I mentioned in another comment, this alternative should be instanciate with an empty list.

Also we should linked this with the Crud like https://github.com/gravitee-io/gravitee-api-management/blob/master/gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/test/java/inmemory/ApiKeyQueryServiceInMemory.java

It will help the test later ;)